source/localizable/object-model/computed-properties.md

```diff
@@ -27,13 +33,53 @@ let ironMan = Person.create({
 ironMan.get('fullName'); // "Tony Stark"
 ```
 
-This declares `fullName` to be a computed property, with `firstName` and `lastName` as the properties it depends on. The
-first time you access the `fullName` property, the function backing the computed property (i.e. the last argument) will
-be run and the results will be cached. Subsequent access of `fullName` will read from the cache without calling the
-function.  Changing any of the dependent properties causes the cache to invalidate, so that the computed function runs
-again on the next access.
+This declares `fullName` to be a computed property, with `firstName` and `lastName` as the properties it depends on.
+The first time you access the `fullName` property, the function backing the computed property (i.e. the last argument) will be run and the results will be cached.
+Subsequent access of `fullName` will read from the cache without calling the function.
+Changing any of the dependent properties causes the cache to invalidate, so that the computed function runs again on the next access.
+
+### Multiple dependents on the same object
+
+In the previous example, the `fullName` computed property dependend on two other properties:
+
+```javascript
+…
+  fullName: Ember.computed('firstName', 'lastName', function() {
+    let firstName = this.get('firstName');
+    let lastName = this.get('lastName');
+    
+    return `${firstName} ${lastName}`;
+  })
+…
+```
+
+We can also use a short-hand syntax called _brace expansion_ to declare the dependents.
+You surround the dependent properties with braces (`{}`), and separate with `,`:
```

Comment:
Can there be space after comma? If not and if it's a silent error, maybe mention it.